web/src/styles/sections/login-page.scss

```diff
@@ -17,13 +17,14 @@
 
   .MuiFormLabel-root {
     color: variables.$base-darkest;
+    font-weight: bold;
   }
-}
 
-.my-acccount-login-text {
-  color: variables.$primary;
+  p {
+    color: variables.$base-dark;
+  }
```

Comment:
note that this is nested under `.email-check-card` and is not just a blanket `p` selector